source/includes/_portfolio.md

```diff
@@ -0,0 +1,265 @@
+# Portfolio Data
+
+With the introduction of Portfolio-level endpoints, users associated with a data partnership can now submit portfolio-level data on behalf of the reporting entities and fill in the Real Estate Assessment in full. Mind that portfolio-level APIs do not serve all the functionalities in the Portal, such as aggregation, validation, and submission; meaning that API users should still make use of the Portal interface for access to full features.
+
+Portfolio data needs to be submitted per indicator, and requires many requests to submit one full assessment. Each portfolio-level variable corresponds to their own endpoint, and all portfolio-level endpoints are situated in `v0`:
+
+`/api/v0/entities/{entity_id}/responses/{response_id}/values/{variable}`
+
+In order to fetch the full list of variables, along with their description, data type, and accepted values, a request to `/api/v0/entities/{entity_id}/responses/{response_id}/values` **PLACEHOLDER** should be made. After performing a GET request to learn the variable's current value and lock version, in most cases a POST requests can be sent in a payload such as like this:
```

Comment:
This will probably be replaced by a public s3 file having the [slimmed down version of the codebook](https://docs.google.com/spreadsheets/d/1LkF3n9PpbBag0nuY8oxFfgPQA81L012IaViC-D-cXEU/edit?pli=1#gid=437903794).